.travis.yml

```diff
@@ -5,11 +5,12 @@ node_js:
 before_install:
   - npm install -g npm@2.13.5
   - npm install -g nsp
+  - npm install -g grunt-cli
```

Comment:
`node_modules/.bin/grunt` is the one that runs them when it's an npm script, doesn't need the global cli